index.js

```diff
@@ -1,18 +1,67 @@
+/* DISREGARD - first test file, see index_2
```

Comment:
You can always [relocate and/or remove](https://git-scm.com/docs/git-rm) unwanted files from Git using the command line. You can also do it right from Github. Just make sure you have the file copied to another local folder if you intend to keep it for the sake of testing/notes. I only mention this because having multiple files in your project repository can make it confusing for those who are grading, as we don't want to accidentally grade the wrong file!

Reply:
noted - thanks Aaron!

Reply:
got it, thanks @aaronlav ! Just to be sure, index.js is to be disregarded (I'll delete it now) and index_2.js is my submission for the project.